modules/andes-core/broker/src/main/java/org/wso2/andes/server/queue/DLCQueueUtils.java

```diff
@@ -131,4 +135,31 @@ public static synchronized void createDLCQueue(String tenantName, String tenantO
             log.info(dlcQueueName + " Queue Created as Dead Letter Channel");
         }
     }
+
+    /**
+     * Add message to DLC
+     *
+     * @param message
+     *         Message to be moved to DLC
+     */
+    public static void addToDeadLetterChannel(QueueEntry message) {
+        ServerMessage amqMessage = message.getMessage();
+        Long messageID = amqMessage.getMessageNumber();
+        String storageQueue = amqMessage.getRoutingKey();
+        AndesRemovableMetadata removableMessage = new AndesRemovableMetadata(messageID, storageQueue, storageQueue);
+
+        List<AndesRemovableMetadata> messageToMoveToDLC = new ArrayList<>();
+        messageToMoveToDLC.add(removableMessage);
+
+        try {
+            log.info("Moving message to Dead Letter Channel. Message ID " + messageID);
```

Comment:
should this be a info log? (suppose in a loaded environment suddenly mb put lots of messages (thousands) to DLC and log will have lots of lines.?
(point is if our thing works correctly, this should be a debug log)